components/organisms/UserSettingsPage/developer-pack-form.tsx

```diff
@@ -55,7 +55,7 @@ const DeveloperPackForm = ({ refreshUser, providerToken }: DeveloperPackFormProp
       <Button
         onClick={handeVerification}
         variant="primary"
-        className="px-4 py-2 w-max bg-light-slate-4 "
```

Comment:
The PR has conflicts with this file. You can undo the changes and get latest from the beta branch as this file no longer exists.